server/src/test/java/com/project/yozmcafe/domain/member/MemberTest.java

```diff
@@ -61,9 +66,8 @@ void removeUnViewedCafeFail() {
         final UnViewedCafe inValidUnViewedCafe = new UnViewedCafe(cafe4, member2);
 
         //when & then
-        assertThatThrownBy(() -> member.removeUnViewedCafe(inValidUnViewedCafe))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("존재하지 않는 내역입니다.");
+        assertThatThrownBy(() -> member.removeUnViewedCafe(inValidUnViewedCafe)).isInstanceOf(
+                IllegalArgumentException.class).hasMessage("존재하지 않는 내역입니다.");
```

Comment:
변경전 개행 
구조가 낫지 않을까요 ??!
```suggestion
assertThatThrownBy(() -> member.removeUnViewedCafe(inValidUnViewedCafe))
.isInstanceOf(IllegalArgumentException.class)
.hasMessage("존재하지 않는 내역입니다.");
```

Reply:
다른부분도 확인 후 반영할게요!👍